02-technical-guides/hierarchies/02-hierarchy-manipulations.ipynb

```diff
@@ -0,0 +1,2029 @@
+{
```

Comment:
<p>I don't see how knowing which one is the default hierarchy helps me.</p><p>Explain somehow what this features means and what added value it brings to the user</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/atoti/pull/890/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Mmm good point. I amended this by adding the description in our docs for <a href="https://docs.atoti.io/latest/api/atoti.Hierarchy.dimension_default.html#atoti.Hierarchy.dimension_default" target="_blank">https://docs.atoti.io/latest/api/atoti.Hierarchy.dimension_default.html#atoti.Hierarchy.dimension_default</a>. Amended statement is now:</p><p> </p><p>"<strong>Every dimensions in the cube has a designated default hierarchy. This is helpful as some UIs support clicking on a dimension (or dragging and dropping it) as a shortcut to add its default hierarchy to a widget.</strong></p><p> </p><p><strong>Calling the&nbsp;</strong><a href="https://docs.atoti.io/latest/api/atoti.Hierarchy.dimension_default.html#atoti.Hierarchy.dimension_default" target="_blank"><strong>dimension_default</strong></a><strong>&nbsp;property of a hierarchy returns&nbsp;</strong><code><strong>True</strong></code><strong>&nbsp;or&nbsp;</strong><code><strong>False</strong></code><strong>&nbsp;for whether the specified hierarchy is the default hierarchy of its parent dimension. For example, if we call of the&nbsp;</strong><code><strong>dimention_default</strong></code><strong>&nbsp;property of our&nbsp;</strong><code><strong>Address</strong></code><strong>&nbsp;hierarchy, we see the following:</strong>"</p><p> </p><p>and also amended a later statement down the line:</p><p> </p><p>"<strong>A new default hierarchy for a dimension can be assigned as well. For example, we can set our&nbsp;</strong><code><strong>Address</strong></code><strong>&nbsp;hierarchy as the default hierarchy for the&nbsp;</strong><code><strong>Customers</strong></code><strong>&nbsp;dimension like so:</strong>"</p><p> </p><p> </p>